opentelemetry-etw-metrics/src/exporter/mod.rs

```diff
@@ -63,32 +66,278 @@ impl Debug for MetricsExporter {
+                                    name: metric.name.clone(),
+                                    description: metric.description.clone(),
+                                    unit: metric.unit.clone(),
+                                    data: Box::new(data::Gauge {
+                                        data_points: vec![data_point.clone()],
+                                    }),
+                                }],
+                            }],
+                        };
+                        resource_metrics.push(resource_metric);
+                    }
+                } else if let Some(gauge) = data.downcast_ref::<data::Gauge<i64>>() {
+                    for data_point in &gauge.data_points {
+                        let resource_metric = ResourceMetrics {
+                            resource: metrics.resource.clone(),
+                            scope_metrics: vec![ScopeMetrics {
+                                scope: scope_metric.scope.clone(),
+                                metrics: vec![Metric {
+                                    name: metric.name.clone(),
+                                    description: metric.description.clone(),
+                                    unit: metric.unit.clone(),
+                                    data: Box::new(data::Gauge {
+                                        data_points: vec![data_point.clone()],
+                                    }),
+                                }],
+                            }],
+                        };
+                        resource_metrics.push(resource_metric);
+                    }
+                } else if let Some(gauge) = data.downcast_ref::<data::Gauge<f64>>() {
+                    for data_point in &gauge.data_points {
+                        let resource_metric = ResourceMetrics {
+                            resource: metrics.resource.clone(),
+                            scope_metrics: vec![ScopeMetrics {
+                                scope: scope_metric.scope.clone(),
+                                metrics: vec![Metric {
+                                    name: metric.name.clone(),
+                                    description: metric.description.clone(),
+                                    unit: metric.unit.clone(),
+                                    data: Box::new(data::Gauge {
+                                        data_points: vec![data_point.clone()],
+                                    }),
+                                }],
+                            }],
+                        };
+                        resource_metrics.push(resource_metric);
+                    }
+                } else {
+                    global::handle_error(MetricsError::Other(format!(
+                        "Unsupported aggregation type: {:?}",
+                        data
+                    )));
+                }
+
+                for resource_metric in resource_metrics {
```

Comment:
the approach used in this PR is extremely inefficient, as it clones the entire metrics and recreates them into a format that the opentelemetry-proto is able to serialize. This is okay for alpha, and will help resolve the size limit issue, but this must be done in a efficient way before stabilizing the component. (that might require support from upstream sdk)

Reply:
Agreed. I'm working on a more performant solution but am proposing this PR as an immediate fix for #104

Reply:
This is okay for an immediate fix.